Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM osrf/ros:humble-desktop-full
+
+# Install dependencies (taken from devcontainer dockerfile)
+RUN DEBIAN_FRONTEND=noninteractive apt-get update -q && \
+    apt-get update -q && \
+    apt-get install -yq --no-install-recommends \
+    lcov \
```

Comment:
@ryan-roche if this is a containerized Spot ROS 2 driver, it needs no test coverage tooling.